docs/README.md

```diff
@@ -43,15 +43,15 @@ cd ~/Arduino/libraries/IRremoteESP8266 && git pull
 ```
 
 ## Contributing
-If you want to [contribute](.github/CONTRIBUTING.md#how-can-i-contribute) to this project, consider:
-- [Reporting](.github/CONTRIBUTING.md#reporting-bugs) bugs and errors
+If you want to [contribute](../.github/CONTRIBUTING.md#how-can-i-contribute) to this project, consider:
```

Comment:
Changes in this file would break the main page links, that is when you don't look at the specific file.

Does any other project have any kind of fix for this, or do we need absolute paths?